nightfall-client/README.md

```diff
@@ -19,79 +19,39 @@ nightfall-client requires a number of services to be present for it to work. The
 instructions explain how to run all of these up, in a similar 'developer' mode with local file
 system bindings.
 
-If you just want to run Nightfall_2 then an easier way is to use
-[nightfall-scripts](https://github.com/EYBlockchain/nightfall-scripts), which does a similar job but
-uses docker images that do not rely on your local file system and therefore requires no other code
-repositories to be downloaded.
+If you just want to run Nightfall_3 then an easier way is to use
+[start-nightfall script](https://github.com/EYBlockchain/nightfall_3), which does a similar job but
+with all necessary services conteainerised.
 
-Check out the [Nightfall_2](https://github.com/EYBlockchain/nightfall_2) meta repository for an
-overview of all the Nightfall_2 services.
+Check out the [Nightfall_3](https://github.com/EYBlockchain/nightfall_3) meta repository for an
+overview of all the Nightfall_3 services.
 
 ## Building and testing nightfall-client
 
-You'll need Node version >=12.18 so that you have ESM support.
+You need to configure a `client.env` file in the root folder of `nightfall-client` with the needed
+variables to run the client.
 
-First, pull the latest version of Timber (these instructions were tested with v3.2.0). This is an
-important step because Docker won't check to see if the 'latest' version you have locally is up to
-date with dockerhub.
-
-```sh
-$ docker pull eyblockchain/timber:latest
 ```
-
-Clone the [nightfall-client](https://github.com/EYBlockchain/nightfall-client) repository. To run
-this code in developer mode, you will also need to clone:
-
-- [zokrates-worker](https://github.com/EYBlockchain/zokrates-worker)
-- [nightfall-deployer](https://github.com/EYBlockchain/nightfall-deployer)
-- [nightfall-timber](https://github.com/EYBlockchain/nightfall-timber)
-
-Make sure you put all of these repositories under a common root in your file system because the
-docker-compose yaml assumes they are accessible via `../<application>`. Then, in the
-`nightfall-client` root directory, locally install the files you'll need for testing via:
-
-```sh
-$ npm i
+ETH_NETWORK=goerli or mainnet
+BLOCKCHAIN_URL=your web3 url provider to access the blockchain
+CIRCUIT_FILES_URL=url of the repository for the circuit files
+CONTRACT_FILES_URL=url of the repository for the contract files
 ```
 
-Build the docker images (you only need do this once unless you makes changes to package.json). To do
-this you will need a personal access token with repo permissions. This is because one of the build
-stages of `zokrates-worker` needs to pull a GPR package:
+To run the script with existing images:
 
-```sh
-$ docker-compose build --build-arg GPR_TOKEN=<paste your token here>
+```
+./start-client -c
```

Comment:
How do you stop a client?

Reply:
With ctrl+c as start-nightfall. It shuts down the containers because the `trap` in the `start-client` script catch the event and runs a docker-compose down.